channels/do.m3u

```diff
@@ -8,9 +8,7 @@ https://5b38ce71f1f00.streamlock.net/8180/8180/playlist.m3u8
 #EXTINF:-1 tvg-id="Canal56.do" tvg-name="Canal 56" tvg-country="DO" tvg-language="" tvg-logo="http://emisoradominicana.net/television/wp-content/uploads/2015/06/tv-recuerdos-56.jpg" group-title="",Canal 56 [Not 24/7]
 https://cloudflare.streamgato.us:3549/live/canal56live.m3u8
 #EXTINF:-1 tvg-id="Canal8PUNTACANATVHD.do" tvg-name="Canal 8 PUNTA CANATV HD" tvg-country="DO" tvg-language="" tvg-logo="https://pbs.twimg.com/profile_images/735482998372630529/bEgiPOU1_400x400.jpg" group-title="",Canal 8 PUNTA CANATV HD
-https://cdn4.hostlagarto.com:8081/static/puntacanatv/chunks.m3u8
```

Comment:
Works

Reply:
Line 11 ;)

Reply:
This is line 11 isn't it? I don't see other Canal8PUNTACANATVHD here

Reply:
The new line 11, not the line before change.